oauth2-server/src/test/java/com/clouway/oauth2/util/ParamRequest.java

```diff
@@ -0,0 +1,85 @@
+package com.clouway.oauth2.util;
+
+import com.clouway.friendlyserve.RequestExt;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author Mihail Lesikov (mlesikov@gmail.com)
+ */
+public class ParamRequest implements RequestExt {
```

Comment:
no, it doe not implements RequestExt, it implement just Request and there was a problem with the type cast  - coud not typecas ParamRequest to RequestExt.
I saw it and would like to use it, but the fserve is in the maven repo  and I have to call you to update and... 

I will add the change in the fserve and we can do the magic